tests/manual-test-cases/Group19-ROBO/19-5-ROBO-Vcenter-Connectivity.md

```diff
@@ -0,0 +1,35 @@
+Test 19-5 - ROBO vCenter Connectivity
+=======
+
+# Purpose:
+To verify that the applications deployed in containerVMs in a ROBO Advanced environment are functional when the ESXi(s) hosting the containerVMs are disconnected from the vSphere host. This test exercises the WAN connectivity and resiliency support for a ROBO environment that could represent a customer's cluster topology. 
+
+# References:
+1. [vSphere Remote Office and Branch Office](http://www.vmware.com/products/vsphere/remote-office-branch-office.html)
+
+# Environment:
+This test requires access to VMware Nimbus cluster for dynamic ESXi and vCenter creation. This test should be executed in the following topologies and should have vSAN enabled.
+* 1 vCenter host with 3 clusters, where 1 cluster has 1 ESXi host and the other 2 clusters have 3 ESXi hosts each
+* 2 vCenter hosts connected with ELM, where each vCenter host has a cluster/host/datacenter topology that emulates a customer environment (exact topology TBD)
+
+See https://confluence.eng.vmware.com/display/CNA/VIC+ROBO for more details.
+
+# Test Steps:
+1. Deploy a ROBO Advanced vCenter testbed for both environments above
+2. Install the VIC appliance on vCenter
```

Comment:
> Install the VIC appliance on vCenter

Install a VCH in a cluster

I'm going to be increasingly pedantic about word usage going forwards, as part of my responsibilities to ensure clear/concise communication. This same comment applies to other uses of this term - VIC appliance is an overloaded term at this time, for example I *think* you're talking about a VCH here, but you may well be talking about testing the VIC appliance with Harbor/Admiral across a WAN link.

Harbor/Admiral over WAN is a good test and we should likely add a section for, even if it's a statement that we're explicitly not testing that facet at this time @cgtexmex .

Reply:
> VIC appliance is an overloaded term at this time

Agree completely. I'm replacing all occurrences of `VIC appliance` with `VCH` in this change. For this particular test, I'll add some steps to test Harbor/Admiral with Engine through the VIC appliance.